streamparser.py

```diff
@@ -158,7 +159,14 @@ def __init__(self, lexical_unit):  # type: (str) -> None
         self.lexical_unit = lexical_unit
 
         cohort = re.split(r'(?<!\\)/', lexical_unit)
-        self.wordform = cohort[0]
+
+        if ']]^' in cohort[0]:
+            self.wordbound_blank, self.wordform = cohort[0].split(']]^', 1)
+            self.wordbound_blank += ']]'
+        else:
+            self.wordbound_blank = ''
```

Comment:
Isn't it possible to have `[[]]`? In that case, you would get the same result, right? It feels like this ought to be `None` here instead. What do you think?

Reply:
If there's no attached wordbound blank to an LU then you should be able to test that it's empty. Technically we'll never have ``[[]]``, but having an empty string in the wordbound blank variable would be consistent with all the changes made in the apertium code.

Reply:
It can be ``None``. But no, ``[[]]`` isn't possible.

Reply:
What do you think provides better semantics/a nicer API here?

Reply:
In case someone wants to concatenate the wordbound blanks with the LU or combine wblanks, it might be better to return an empty string rather than ``None``. But I don't use python much so you'd be a better judge of that.

Reply:
That's a good point. How does other apertium tooling behave in the presence of `[[]]^...`?

I generally prefer having empty string and `None` having different semantics (i.e. this should be `= None` IMO) but it's not a strongly held opinion in this particular case.

Reply:
With other tools, the wordbound_blank member is an empty string if there's no attached wblank. I have no strong opinion here either, mostly I think it won't make much of a difference, and if they wanna concatenate and stuff they can check for ``None`` first anyway, so I'll just change it.

Reply:
hmmm I can't just change it to ``None``?

Reply:
Ok I've decided to keep it as an empty string. It'll be much more helpful and consistent with other tools.